### PR TITLE
API endpoint to list project contributions by day (rebased)

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This is a rebased PR with latest develop. Same approach as #1681

### CODE REVIEW

File: server/services/project_service.py [hunk 1]
```diff
@@ -35,6 +38,36 @@ def get_project_by_id(project_id: int) -> Project:
...
+        .group_by(TaskHistory.action, 'date')\
+        .order_by(TaskHistory.action, 'date')\
+        .all()
+
+        contribs_dto = ProjectContribsDTO()
+
+        if len(results) == 0:
+            raise NotFound()
+
+        mapped_contribs = [ProjectContribDTO(dict(date=str(r[1]), total=r[2])) for r in results
```
Comment (willemarcel) [inline server/services/project_service.py]: I would prefer to have the data on this format:
```
[
     {"date": "2019-04-08",
      "mapped": 4,
      "validated": 0
    },
    {"date": "2019-04-09",
      "mapped": 1,
      "validated": 5
    }
]
```

**Comment by JorgeMartinezG** [comment]
@willemarcel I've implemented requested changes. Please verify again  👍

**Comment by xamanu** [comment]
@JorgeMartinezG Can you please reroll on latest `develop`? Thanks.

**Comment by JorgeMartinezG** [comment]
@xamanu this branch has been rebased with latest develop and it is ready (I think) 👍

**Comment by willemarcel** [comment]
Thanks, @JorgeMartinezG !

**Pull request merged**